changelogs/fragments/20240611-helm-rc-version.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - helm - Helm version checks did not support RC versions. They now accept any version tags. (https://github.com/ansible-collections/kubernetes.core/pull/745).
```

Comment:
Please add [new_line_at_end_of_file](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.new_line_at_end_of_file)

Reply:
I think there is one, GH perhaps don't show it :

```
% cat -e changelogs/fragments/20240611-helm-rc-version.yaml
bugfixes:$
  - helm - Helm version checks did not support RC versions. They now accept any version tags. (https://github.com/ansible-collections/kubernetes.core/pull/745).$
```

```
% yamllint changelogs/fragments/20240611-helm-rc-version.yaml && echo 'ok'
ok
```

Lint passed too. Am I missing something ?

Reply:
Probably I missed it in review, but I did see it [here](https://github.com/gileri/kubernetes.core/blob/helm-version-rc/changelogs/fragments/20240117-fix-helm-diff-cmd-line-rendering.yml). Anyway, it's ok if linter passed

Reply:
Github does display an explicit symbol when there is **no** newline at end-of-file.

Reply:
> Github does display an explicit symbol when there is **no** newline at end-of-file.

got it, thanks